stat_handler.go

```diff
@@ -119,11 +134,32 @@ func (h *StatHandler) WorkerResumeHandler() func(http.ResponseWriter, *http.Requ
 	}
 }
 
+// ScoreBoardHandler returns http.HandlerFunc implementation for /scoreboard request
+func (h *StatHandler) ScoreBoardHandler() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if !reqMethodValidate(w, r, "GET") {
+			return
+		}
+
+		busy := len(h.Tracker.queueStack)
+
+		renderJSON(w, &ScoreBoardResponse{
+			TotalHandled:   int(h.Tracker.ScoreBoard.TotalSucceeded + h.Tracker.ScoreBoard.TotalFailed),
```

Comment:
This logic of calculation is concern of `ScoreBoard`.
implement this as `ScoreBoard` method.